Update README file with references to externals (#680)
HALFpipe/HALFpipe: README.rst

group analysis. It provides state-of-the-art preprocessing using
`fmriprep <https://fmriprep.readthedocs.io/>`__, but removes the
necessity to convert data to the `BIDS
<https://bids-specification.readthedocs.io/en/stable/>`__ format.

> Tools from well-established neuroimaging software packages are combined either directly our through our dependencies,
> such as `ANTs <https://antspy.readthedocs.io/>`__ and `FreeSurfer <https://surfer.nmr.mgh.harvard.edu/>`__ for preprocessing,
> and `FSL <http://fsl.fmrib.ox.ac.uk/>`__ and `nipype <https://nipype.readthedocs.io/>`__ for calculating common resting-state and task-based fMRI features.
> We strongly urge users to acknowledge these tools when publishing results obtained with HALFpipe.

Reviewer: ```suggestion
HALFpipe relies on tools from well-established neuroimaging software packages, either directly or through our dependencies,
including as `ANTs <https://antspy.readthedocs.io/>`__ and `FreeSurfer <https://surfer.nmr.mgh.harvard.edu/>`__ for preprocessing,
and `FSL <http://fsl.fmrib.ox.ac.uk/>`__ and `nipype <https://nipype.readthedocs.io/>`__ for calculating common resting-state and task-based fMRI features.
We strongly urge users to acknowledge these tools when publishing results obtained with HALFpipe.
```

> <https://bids-specification.readthedocs.io/en/stable/>`__ format.
> 
> HALFpipe relies on tools from well-established neuroimaging software packages, either directly or through our dependencies,
> including as `ANTs <https://antspy.readthedocs.io/>`__ and `FreeSurfer <https://surfer.nmr.mgh.harvard.edu/>`__ for preprocessing,
> and `FSL <http://fsl.fmrib.ox.ac.uk/>`__ and `nipype <https://nipype.readthedocs.io/>`__ for calculating common resting-state and task-based fMRI features.
> We strongly urge users to acknowledge these tools when publishing results obtained with HALFpipe.

Reviewer: ```suggestion
<https://bids-specification.readthedocs.io/en/stable/>`__ format. Common
resting-state and task-based fMRI features can then be calculated on the
fly.

`HALFpipe` relies on tools from well-established neuroimaging software 
packages, either directly or through our dependencies, including `ANTs 
<https://antspy.readthedocs.io/>`__, `FreeSurfer 
<https://surfer.nmr.mgh.harvard.edu/>`__,  `FSL <http://fsl.fmrib.ox.ac.uk/>`__ 
and `nipype <https://nipype.readthedocs.io/>`__. We strongly urge users to 
acknowledge these tools when publishing results obtained with HALFpipe.
```

Reviewer: I'm sorry, reading it again made me think that we don't use FSL just for calculating features, but also for preprocessing, so wanted that to be precise
